cmd/queue/main.go

```diff
@@ -187,6 +191,10 @@ func main() {
 		flush(logger)
 		os.Exit(1)
 	case <-ctx.Done():
+		// If container-freezer enabled, call terminiating function
```

Comment:
nit: this comment doesn't really add anything over the code, and it'd be nice not to have the concept of "container-freezer" mentioned in QP (even in the comment) since we're trying to keep it a little generic